doc/input-output-reference/src/overview/group-system-availability-managers.tex

```diff
@@ -628,6 +637,16 @@ \subsubsection{Outputs}\label{outputs-9-008}
 
 HVAC,Average,Availability Manager Hybrid Ventilation Control Status {[]}
 
+HVAC,Average,Hybrid Ventilation Control HVAC System Opration Elapsed Time {[min]}
```

Comment:
@mjwitte Thanks for pointing typos. I corrected.

Regarding your concern of output variables of elapsed time, there is an existing output variable to report control status with 0, 1, 2: Availability Manager Hybrid Ventilation Control Status. If I just report 0 or 1, we don't need these two new output variables. I can remove them. Please advice.